src/widgets/WebexMeeting/WebexMeeting.jsx

```diff
@@ -25,27 +25,38 @@ export default class WebexMeetingWidget extends Component {
     });
     this.state = {
       adapterConnected: false,
+      meeting: null,
     };
     this.adapter = new WebexSDKAdapter(webex);
+    this.meetingSubscription = null;
   }
 
   async componentDidMount() {
     await this.adapter.connect();
     // Once adapter connects, set our app state to ready.
     this.setState({adapterConnected: true});
+
+    this.meetingSubscription = this.adapter.meetingsAdapter
```

Comment:
Let's follow the observable name pattern
```suggestion
    this.meeting$ = this.adapter.meetingsAdapter
```

Reply:
we don't really need the observable itself here, we store just the subscription to it (so we can unsubscribe later). I was thinking of naming it `meetingSub` for brevity, but then there is a potential confusion with the term "subject".

I also saw a pattern of having subscriptions in a component indexed by name in a `this.subscriptions` object, it's a bit more generic than we currently need here but it could work:

```js
// constructor
this.subscriptions = {};

// didMount
this.subscriptions.meeting = obs.subscribe(...);

// willUnmount
for(let sub of Object.values(this.subscriptions)) {
  sub.unsubscribe();
}
```

Reply:
My bad, I only saw the first line and didn't realize the rest of the operators in the following lines 😅

Actually, the most efficient way to dispose of all subscriptions is to use the `add()` operator of a `Subscription` and unsubscribe from the subscription that hold them all.
See https://rxjs-dev.firebaseapp.com/guide/subscription and how we use it [in our integration tests](https://github.com/webex/sdk-component-adapter/blob/master/src/RoomsSDKAdapter.integration.test.js#L88)